finality-aleph/Cargo.toml

```diff
@@ -11,14 +11,14 @@ repository.workspace = true
 # fixed version to 'freeze' some types used in abft, mainly `SignatureSet` used in justification and signature aggregation
 aleph-bft-crypto = { workspace = true }
 
-current-aleph-bft = { package = "aleph-bft", version = "0.20" }
-current-aleph-bft-rmc = { package = "aleph-bft-rmc", version = "0.6" }
-legacy-aleph-bft = { package = "aleph-bft", version = "0.19" }
-legacy-aleph-bft-rmc = { package = "aleph-bft-rmc", version = "0.5" }
+current-aleph-bft = { package = "aleph-bft", version = "0.33" }
+current-aleph-bft-rmc = { package = "aleph-bft-rmc", version = "0.11" }
+legacy-aleph-bft = { package = "aleph-bft", version = "0.20" }
+legacy-aleph-bft-rmc = { package = "aleph-bft-rmc", version = "0.6" }
 
 network-clique = { workspace = true }
 primitives = { workspace = true }
-legacy-aleph-aggregator = { package = "aggregator", git = "https://github.com/Cardinal-Cryptography/aleph-node.git", tag = "aggregator-v0.2.1" }
+legacy-aleph-aggregator = { package = "aggregator", git = "https://github.com/Cardinal-Cryptography/aleph-node.git", tag = "r-12.0" }
```

Comment:
`current` aggregator has to become legacy. Using tag from release 12 here for the new legacy aggregator - probably better to create a new tag on the current `main` branch called `aggregator-something`

Reply:
yep, its not the greates dev-experience but pls prepare pr which bumps the aggregator version and then create a tag for that

Reply:
Hm, can't I just create a tag on the current `main`?

Reply:
Actually, I think it's best to create a tag for aggregator at the last commit which altered it. This would be A0-3009: (&A0-3020) Timed block metrics (#1411)

Reply:
Sorry, are we talking about creating a tag in `aleph-node`? We should be doing that for release-thing and only on the release branches

Reply:
If we're tatlking about `aleph-bft` repo, maybe it's the time to tackle release cycle process in that repo

Reply:
Oh, I've already created it [here](https://github.com/Cardinal-Cryptography/aleph-node/releases/tag/aggregator-v0.5.0). I saw there are some tags in the past dedicated to aggregator, so I did it in analogy to them.

After some thought, I believe we could use the `r12` tag for aggregator, or create `aggregator-v0.4.0` tag equivalent to `r12`.

Reply:
I removed `aggregator-v0.5.0` and introduced `aggregator-v0.4.0` which is the same as `r-12.0`

Reply:
At the end, removed custom aggregator tag and used r-12.1 tag.

Reply:
cool thanks